components/screens/DailyRidesScreen/DailyRidesStyleSheet.js

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet } from 'react-native';
```

Comment:
Super small change.
I realize that you were following the naming conventions that I already started. But I have found that `[Name]ScreenStyles.js` is a little nicer to read and fits within the CSS conventions a little more closely.

A small change I know, but I've started implementing this naming convention on my style sheets. a simple rename changing "...StyleSheets.j" to "..Styles.js".

Reply:
I renamed my stylesheet from DailyRidesStyleSheet.js to DailyRidesSrceenStyles.js.